test/key_test.js

```diff
@@ -721,3 +721,43 @@ QUnit.test("typing in an input type=number works", function() {
 		start();
 	});
 });
+
+
+QUnit.test("Key property, a typed", function () {
+	stop();
+
+	var a = false;
+	st.binder('key', 'keypress', function (ev) {
+		a = ev.key;
+		equal('a', ev.key);
+	});
+	syn.type('key', "a", function () {
+		ok(a, "a key typed");
+		start();
+	});
+});
+
+QUnit.test("ctrl key", function () {
+	stop();
+
+	var keyIsDown = false;
+	st.binder("key", "keydown", function (ev) {
+		keyIsDown = ev.ctrlKey;
+		ok(ev.key === 'ctrl', "key is normalized");
```

Comment:
This doesn’t look right to me. According to [these docs](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values), I think it should be `"Control"`.

Reply:
Thank you for the catch @chasenlehara, I missed this docs.
It is `"Control"` for sure.
<img width="867" alt="Screen Shot 2020-03-26 at 5 29 54 PM" src="https://user-images.githubusercontent.com/109013/77671205-8ae8c480-6f87-11ea-966a-aa1762ae4a5a.png">

Reply:
I updated the code @chasenlehara !